test/suite/wttest.py

```diff
@@ -177,6 +177,10 @@ def finalReport():
     def currentTestCase():
```

Comment:
I would change to getCurrentTestCase() to adhere to get/set lingo.

Reply:
Makes sense. See [e76b17c](https://github.com/wiredtiger/wiredtiger/pull/12357/commits/e76b17ca3ef9ad14af99a07e689f13cf2501bdcc)